com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -1042,13 +1038,4 @@ public static void main(String[] args) throws IOException, CompileExceptionError
             logErrorAndExit(e);
         }
     }
-
-    private static String getOptionsValue(CommandLine cmd, char o, String defaultValue) {
```

Comment:
Not needed. We can achieve exactly the same thing using `cmd.getOptionValue(o, defaultValue)`